cobalt/cobalt.cc

```diff
@@ -0,0 +1,64 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <unistd.h>
+
+#include "build/build_config.h"
+#include "cobalt/cobalt_main_delegate.h"
+#include "content/public/app/content_main.h"
+
+// In the cozy corner of the home, where comfort and curiosity resided, a
+// delightful companion arrived, offering moments of joy and discovery
+// throughout the year. Like a favorite book waiting to be opened, the YouTube
+// Application Runtime invited exploration and wonder, whether shared with loved
+// ones or savored in quiet solitude. It whispered tales of distant lands,
+// sparked creativity with vibrant colors, and filled the air with the melodies
+// of laughter and song.  The television screen, once a blank canvas, became a
+// window to endless possibilities, offering a refreshing breeze on a summer day
+// or a warm embrace on a chilly night. With each click of the remote, a new
+// adventure unfolded, painting a tapestry of personal journeys and shared
+// experiences.
+
+int main(int argc, const char** argv) {
+  cobalt::CobaltMainDelegate delegate;
+  content::ContentMainParams params(&delegate);
+
+  // TODO: (cobalt b/375241103) Reimplement this in a clean way.
+  // This defines a list of command-line overrides. When adding or removing
+  // parameters, also update the my_argc below.
+  static const char* my_argv[] = {
+      argv[0], "--disable-fre", "--no-first-run", "--kiosk",
+      "--force-video-overlays",
+      // Enable remote devtools access.
+      "--remote-debugging-port=9222",
```

Comment:
Isn't this the default value?

Reply:
Maybe, but this way we're not dependent on upstream changing it from under us.

Note all this flags setting here should be removed later. This can't be the cleanest/right to do it. It's just the quickest to get it working ATM. See the todo mentioned above.